automated-comments/java/hamming/calculate_distance_in_constructor.md

```diff
@@ -0,0 +1,3 @@
+This solution seems to be calculating the Hamming distance inside the `getHammingDistance()` method.
+While this is acceptable, it means that the calculation might be done again if you did not use lazy initialization.
+Consider calculating the Hamming distance inside the constructor and storing the result in a member variable which can be returned inside `getHammingDistance()`.
```

Comment:
potentially start with this. From the [guidelines](https://github.com/exercism/automated-analysis/blob/5b01bae1faced4c771308bab10a4fd6277e2a4d1/docs/comment_copy_guidelines.md):

> When possible, put the recommendation first, then the explanation.